specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2024-06-01-preview/examples/ListDiagnostics.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "eastus"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "x-ms-request-id": "27b7c568-16ec-46f3-bcf1-5bea3f2529b1"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus/diagnostics/diskInspection",
+            "name": "diskInspection",
+            "type": "Microsoft.Compute/locations/diagnostics",
+            "properties": {
+              "supportedResourceTypes": [
+                "Microsoft.Compute/virtualMachines"
+              ]
+            }
+          },
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/{location}/diagnostics/spotPlacementRecommender",
```

Comment:
Please change these as well to the new API paths - this API response will need to change in the CDRP repo code as well

Reply:
Awaiting guidance on existing path deprecation. I am making this as an additive change for now. I have added new entry for placementScores/spot/generate in ListDiagostics.json.

Reply:
As per guidance received, we will carry forward the old path in the new version and delete them in future versions once we are able to demonstrate no traffic to the old paths.